frontend/src/components/Project/GMS/GMSForm.js

```diff
@@ -75,7 +75,7 @@ const GMSForm = () => {
         />
       </div>
       <div className="form-group form-section-title">
-        Exceedance rate level
+        Exceedance rate level (/years)
```

Comment:
Is the / meant to be there?

Reply:
I think this one is a bit tricky, he specifically said `/years` for the annual exceedance rate. But, I might probably double-check with Brendon on Monday whether we want to keep `Exceedance rate level` and use `/years` as a unit or rename it to Return Period.

Reply:
It's technically years<sup>-1</sup> so if you can do the subscript that's probably more clearer. Otherwise `/years` is appropriate.

Reply:
By looking at the Vs30 in the site selection tab, the subscript was doable. Hence, superscript must be doable.